src/FixedDataTableCellDefault.js

```diff
@@ -64,7 +64,7 @@ var FixedDataTableCellDefault = React.createClass({
   },
 
   render() {
-    var {height, width, style, className, children, columnKey, ...props} = this.props;
+    var {height, width, style, className, children, columnKey, rowIndex, ...props} = this.props;
```

Comment:
Interesting destructing to remove props... 
Confused me for a second so might be worth adding a comment as to why it's done.  Maybe this is just a common pattern I haven't seen yet though.